consensus/istanbul/core/roundstate.go

```diff
@@ -29,19 +29,26 @@ import (
 // newRoundState creates a new roundState instance with the given view and validatorSet
 func newRoundState(view *istanbul.View, validatorSet istanbul.ValidatorSet, preprepare *istanbul.Preprepare, pendingRequest *istanbul.Request, preparedCertificate istanbul.PreparedCertificate, hasBadProposal func(hash common.Hash) bool) *roundState {
 	return &roundState{
-		round:               view.Round,
-		desiredRound:        view.Round,
-		sequence:            view.Sequence,
-		Preprepare:          preprepare,
-		Prepares:            newMessageSet(validatorSet),
-		Commits:             newMessageSet(validatorSet),
+		round:        view.Round,
+		desiredRound: view.Round,
+		sequence:     view.Sequence,
+		Preprepare:   preprepare,
+		Prepares:     newMessageSet(validatorSet),
+		Commits:      newMessageSet(validatorSet),
+		// todo not necessarily this validator set, might be the parent ones
+		ParentSeals:         newMessageSet(validatorSet),
```

Comment:
Should we call this `ParentCommits` for consistency? Technically the `seal` is included in the `Commit` message, and it looks like we're storing the whole message here